Doppler.Currency.Test/BnaHandlerTests.cs

```diff
@@ -93,7 +93,7 @@ public async Task GetCurrency_ShouldBeReturnUsdCurrencyOfBna_WhenHtmlHaveTwoCurr
 
             var result = await service.GetCurrencyByCurrencyCodeAndDate(dateTime, CurrencyCodeEnum.Ars);
 
-            Assert.Equal($"{dateTime:yyyy/MM/dd}", result.Entity.Date);
+            Assert.Equal(dateTime.ToUniversalTime(), result.Entity.Date);
```

Comment:
In my opinion you should use hardcoded dates in the assertions.

Reply:
Done